tests/test_version.py

```diff
@@ -14,4 +14,4 @@
 
 # This test is primarily intended to ensure the test suite itself is working.
 def test_version():
-    assert version("myna") == "0.1.0.dev0"
+    assert version("myna") == "1.1.0.dev0"
```

Comment:
The intent of this is to check the version of the installed package against the version of the pyproject.toml package. This could be dynamically checked instead of hardcoded:

```python
with open(os.path.join("..", "pyproject.toml") as f:
   lines = f.readlines()

toml_version = lines[["version = " in l for l in lines].index(True)].split(" = ")[-1].split('"')[1]
assert (version("myna") == toml_version
```

Reply:
Added - this makes more sense. No reason to forget to do this again..